src/content/cookbook/forms/validation.md

```diff
@@ -149,7 +149,7 @@ ElevatedButton(
 ### How does this work?
 
 To validate the form, use the `_formKey` created in
-step 1. You can use the `_formKey.currentState()`
+step 1. You can use the `_formKey.currentState`
 method to access the [`FormState`][],
```

Comment:
You should probably update `method` to `get accessor` or similar?

Reply:
do you mean to get `fields` of `FormState` which are private? There was some discussion of making `fields` public.
https://github.com/flutter/flutter/issues/67283#issuecomment-712220425

Reply:
No, you have retained the old text that referred to `currentState` as a `method`. It's probably better referred to as an `accessor` instead.

Reply:
Oh yes, correcting it as well.

Reply:
@domesticmouse fixed it.